### DISCUSSION TITLE
What is "Other" in latency page?

### DISCUSSION THREAD

**Discussion by szostid** [alvr-org/ALVR#564] [Q&A]
Every time im playing ALVR about 60% of my latency comes from "other" - what it actually is? Can't find any answers here.

**Comment by chiwangnvp** [comment]
"totalLatency": 184.666, 
"receiveLatency": 68.770, "renderTime": 18.537, "idleTime": 0.297, "waitTime": 11.727, "encodeLatency": 4.892, "sendLatency": 76.696, "decodeLatency": 3.748
